src/util/net/util.h

```diff
@@ -0,0 +1,43 @@
+#ifndef HEADER_fd_src_util_net_util_h
+#define HEADER_fd_src_util_net_util_h
+
+#include "fd_udp.h"
+#include "fd_eth.h"
+
+typedef struct __attribute__((packed)) {
+  fd_eth_hdr_t eth[1];
+  fd_ip4_hdr_t ip4[1];
+  fd_udp_hdr_t udp[1];
+} eth_ip_udp_t;
+
+FD_PROTOTYPES_BEGIN
+
+void
```

Comment:
```suggestion
eth_ip_udp_t *
```
It should probably return pkt.

Reply:
Also needs a doc comment